### ISSUE TITLE
Create a default class

### ISSUE THREAD

**Issue by guillaume-chevalier** [Neuraxio/Neuraxle-PyTorch#2]
The class must have the following arguments in the constructor, and if None we will use some default values: 
- A wrapped `nn.Module`
- Criteria (loss function)
- Optimizer (e.g.: if None, use Adam by default)

I'm also tempted to add those arguments, but I'm not sure: 
- Optimizer's hyperparams
- `nn.Module`'s hyperparams

Before starting, let's think at how we accumulate statistics during training. E.g.: might need to pass a list of metrics, and perhaps callback functions or something for validation-time evaluation of the model for early stopping for instance. 

TODO: Might need to add the early stopping and the related things such as the list of metrics instead in a ValidationWrapper. Needs more design before coding.
